examples/fluids/index.rst

```diff
@@ -272,16 +272,41 @@ of total energy, is given by
    :label: eq-advection
 
 with :math:`\bm{u}` the vector velocity field. In this particular test case, a
-blob of total energy (defined by a characteristic radius :math:`r_c`) is transported by
-a uniform circular velocity field. We have solved :math:numref:`eq-advection`
-with zero energy density :math:`E`, and no-flux for :math:`\bm{u}`.
-The :math:`3D` version of this problem can be run with::
+blob of total energy (defined by a characteristic radius :math:`r_c`) is transported by two
+different wind types.
 
-   ./navierstokes -problem advection
+- **Rotation**
 
-while the :math:`2D` version with::
+   In this case, a uniform circular velocity field transports the blob of total energy. We have
+   solved :math:numref:`eq-advection` with zero energy density :math:`E`, and no-flux for :math:`\bm{u}`.
 
-   ./navierstokes -problem advection2d
+   The :math:`3D` version of this test case can be run with::
+
+      ./navierstokes -problem advection -problem_advection_wind rotation
+
+   while the :math:`2D` version with::
+
+      ./navierstokes -problem advection2d -problem_advection_wind rotation
+
+- **Translation**
+
+   In this case, a background wind with a constant velocity field, enters the domain and compels
+   the blob of total energy to leave the domain.
```

Comment:
```suggestion
   In this case, a background wind with a constant rectilinear velocity field, enters the domain and transports
   the blob of total energy out of the domain.
```

Reply:
I suggested `rectilinear` just to be extra clear, since constant alone does not specify this

Reply:
Oh, sorry I missed it.